homeassistant/components/neato/strings.json

```
@@ -13,7 +13,7 @@
     },
     "error": {
       "invalid_credentials": "Invalid credentials",
-      "unexpected_error": "Unexpected error"
+      "unexpected_error": "[%key:common::config_flow::error::unknown%]"
```

Would be nice to rename "unexpected_error" to unknow also in the key and update config flow + tests

---

Is your suggestion to rename in the homeassistant/strings.json "unknown": "Unexpected error" -> "unexpected": "Unexpected error"
Because in the base homeassistant/strings.json unknown currently translates to Unexpected error

---

```suggestion
      "unknown": "[%key:common::config_flow::error::unknown%]"
```
Here is what i suggest :) But with this, you'll have to update tests and config_flow.py

---

Could be in another PR as it's not the main goal of this one.

---

Good idea, I see that other files also name it unknown, I'll try to figure out how to change this and create a separate PR for this

---

We want the keys to also be replaced in the same PR as where the values are changed but only one integration at a time.

---

Nice, so I can do the key and reference change in the same PR for only neato component

---

Yes, please. 👍